DEVELOPMENT.md

```diff
@@ -50,36 +50,10 @@ your laptop.
 Create a GitHub personal token with `admin:repo_hook` persmissions
 at least. This will be used to create the webhook.
 
+Run blow script to deploy Tekton CI on the local kind cluster:
```

Comment:
NIT: Run the script below to ...

Reply:
The comments about the meaning of the params have been removed, it would be nice to have them in here, something like (but using the parameter names):
```
GITHUB_ORG=<org> # The org or user where your fork is hosted	
GITHUB_REPO=<repo> # The name of the fork, typically "plumbing"	
GITHUB_USER=<github-user> # Your GitHub username`
```

Reply:
@afrittoli thank you for the comments, I committed new change, please review.